benchmarks/src/main/scala/zio/IONarrowFlatMapBenchmark.scala

```diff
@@ -112,10 +112,10 @@ class IONarrowFlatMapBenchmark {
 
   private[this] def zioNarrowFlatMap(runtime: Runtime[Any]): Int = {
     def loop(i: Int): UIO[Int] =
-      if (i < size) IO.succeedLazy[Int](i + 1).flatMap(loop)
-      else IO.succeedLazy(i)
+      if (i < size) IO.succeed[Int](i + 1).flatMap(loop)
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
:heavy_check_mark: